client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/EventShowingsContainer.tsx

```diff
@@ -0,0 +1,164 @@
+    case 6:
+      return (a, b) => b.totalseats - a.totalseats;
+    case 7:
+      return (a, b) => b.availableseats - a.availableseats;
+    default:
+      return (a, b) =>
+        new Date(toDateStringFormat(a.eventdate)).getTime() -
+          new Date(toDateStringFormat(b.eventdate)).getTime();
+    }
+  };
+
+  return (
+    <div
+      className={
+        'bg-white flex flex-col mt-6 p-6 rounded-xl shadow-xl text-zinc-800'
+      }
+    >
+      <div className={'grid grid-cols-12 p-2'}>
+        <h2
+          className={
+            'col-span-12 min-[650px]:col-span-6 text-xl md:text-2xl text-center min-[650px]:text-start pb-1'
+          }
+        >
+          {eventData?.eventname} Showings
+        </h2>
+        <div
+          className={
+            'col-span-12 min-[650px]:col-span-6 flex flex-row justify-center min-[650px]:justify-end'
+          }
+        >
+          <div className={'col-span-6 grid min-[1350px]:grid-cols-2 pr-1'}>
+            <label
+              htmlFor={'sortMethod'}
+              className={
+                'text-xs text-zinc-800 my-auto pr-2 text-center min-[1350px]:text-end'
+              }
+            >
+              Sort By:
+            </label>
+            <select
+              disabled={editing}
+              value={sortBy}
+              onChange={(event) => setSortBy(Number(event.target.value))}
+              className={`h-fit m-auto text-xs border border-zinc-500 rounded-xl mr-2 ${
+                showPopUp ? 'hidden' : ''
+              }`}
+            >
+              <option value={0}>Date - Ascending</option>
+              <option value={1}>Showing ID - Ascending</option>
+              <option value={2}>Total Tickets - Ascending</option>
+              <option value={3}>Available Tickets - Ascending</option>
+              <option value={4}>Date - Descending</option>
+              <option value={5}>Showing ID - Descending</option>
+              <option value={6}>Total Tickets - Descending</option>
+              <option value={7}>Available Tickets - Descending</option>
+            </select>
+          </div>
+          <button
+            className={
+              'border border-green-900 bg-green-700 hover:bg-green-800 w-fit h-fit my-auto ' +
+              'disabled:bg-gray-600 text-white p-1 rounded-xl col-span-6 flex flex-row'
+            }
+            onClick={() => {
+              setAdd((add) => !add);
+              setEditing((edit) => !edit);
+            }}
+            disabled={!eventData}
+            aria-label={'Add Showing'}
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              fill='none'
+              viewBox='0 0 20 20'
+              strokeWidth='1.5'
+              stroke='currentColor'
+              className='w-5 h-5 my-auto pr-1'
+            >
+              <path
+                strokeLinecap='round'
+                strokeLinejoin='round'
+                d='M12 6v12m6-6H6'
```

Comment:
Is this icon from hero icons?

Reply:
Yeah, all Icons on the page should be from HeroIcons.